plugin/lighthouse/handler.go

```diff
@@ -55,6 +57,10 @@ func (lh *Lighthouse) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns
 	}
 	serviceIp := ipList[0]
 
+	if state.QType() == dns.TypeAAAA {
+		return lh.emptyIpv6Response(state)
```

Comment:
```suggestion    
                log.Debugf("Returning empty response for TypeAAAA query")
		return lh.emptyIpv6Response(state)
```

Reply:
Done. Had to do manually coz of spacing.